napari/components/viewer_model.py

```diff
@@ -502,9 +506,10 @@ def _on_add_layer(self, event):
 
         if len(self.layers) == 1:
             self.reset_view()
-            ranges = self.layers._ranges
-            midpoint = [self.rounded_division(*_range) for _range in ranges]
-            self.dims.set_point(range(len(ranges)), midpoint)
+            new_point = [
+                rounded_division(*_range) for _range in self.dims.range
```

Comment:
```suggestion
                rounded_division(*range_) for range_ in self.dims.range
```

In python, a variable name with `_` as a first character is standard for marking a variable as unused.